docs/source/cpp/compute.rst

```diff
@@ -1639,17 +1639,19 @@ in the respective option classes.
 Structural transforms
 ~~~~~~~~~~~~~~~~~~~~~
 
-+---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
-| Function name       | Arity      | Input types                         | Output type      | Options class                | Notes  |
-+=====================+============+=====================================+==================+==============================+========+
-| list_element        | Binary     | List-like (Arg 0), Integral (Arg 1) | List value type  |                              | \(1)   |
-+---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
-| list_flatten        | Unary      | List-like                           | List value type  |                              | \(2)   |
-+---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
-| list_parent_indices | Unary      | List-like                           | Int64            |                              | \(3)   |
-+---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
-| struct_field        | Unary      | Struct or Union                     | Computed         | :struct:`StructFieldOptions` | \(4)   |
-+---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
++---------------------+------------+-------------------------------------+------------------+---------------------------------+--------+
+| Function name       | Arity      | Input types                         | Output type      | Options class                   | Notes  |
++=====================+============+=====================================+==================+=================================+========+
+| list_element        | Binary     | List-like (Arg 0), Integral (Arg 1) | List value type  |                                 | \(1)   |
++---------------------+------------+-------------------------------------+------------------+---------------------------------+--------+
+| list_flatten        | Unary      | List-like                           | List value type  |                                 | \(2)   |
++---------------------+------------+-------------------------------------+------------------+---------------------------------+--------+
+| list_parent_indices | Unary      | List-like                           | Int64            |                                 | \(3)   |
++---------------------+------------+-------------------------------------+------------------+---------------------------------+--------+
+| map_array_lookup    | Unary      | Map                                 | Computed         | :struct:`MapArrayLookupOptions` | \(4)   |
```

Comment:
I'm really sorry to bikeshed here, but now that I look at it…would "map_lookup" be a better kernel name? I think we only use "array" in a kernel name when the kernel only works on arrays, but this works on arrays and scalars.

Reply:
Not a problem, better to be rigorous here than have an inconsistent codebase. We just will have to be on the lookout for something that I've missed renaming.